events/event/views.py

```diff
@@ -34,6 +34,31 @@ def get(self, request, pk=None):
         else:
             return HttpResponseForbidden('Permission denied')
 
+    def post(self, request):
+        try:
+            event_data = json.loads(request.body.decode())
+        except Exception as e:
+            return JsonResponse({
+                "json error": e.message,
```

Comment:
@sbufan, Not the best way of handling exception, but even in this case, you should use something like this:
`"json error": str(e),`